source/IAccessibleHandler/orderedWinEventLimiter.py

```diff
@@ -75,10 +75,12 @@ def addEvent(
 		self._genericEventCache[(eventID, window, objectID, childID, threadID)] = next(self._eventCounter)
 		return True
 
-	def flushEvents(self):
+	def flushEvents(self, alwaysAllowedObjects=None):

```

Comment:
> Should this check for None in tuple elements? A test to show what is expected in this case would be good.
I don't understand what you mean here. Do you mean if one of the elements in one of the tuples is None? E.g. windowHandle is None? In that case, specific events would simply not match on that (poorly defined) object...

Reply:
>Do you mean if one of the elements in one of the tuples is None?

Yes, one or more.

> In that case, specific events would simply not match on that (poorly defined) object...

That's fine. A test helps to document that it is an expected arg value, and the behavior in this case is intentional.

I got thinking about this because of this line: 

https://github.com/nvaccess/nvda/blob/1d5e5cee754998906d2cdc98075f06df476da55d/source/IAccessibleHandler/__init__.py#L868